src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -335,11 +332,32 @@ public void setReadTimeout(int timeoutMillis) {
     }
 
     /**
-     * Sets the connect and read timeout to a interactively reasonable duration defined in http.outcall.interactive.*
+     * Sets the connect and read timeout to the values specified in the config block http.outcall.timeouts.*
+     * where * equals the configKey parameter.
+     * <p>
+     * See the http.outcall.timeouts.soap block in component-kernel.conf for reference.
+     *
+     * @param configKey the config key of the timeout configuration block
+     * @return this for fluent method calls
      */
-    public void setInteractive() {
-        setConnectTimeout((int) interactiveConnectTimeout.toMillis());
-        setReadTimeout((int) interactiveReadTimeout.toMillis());
+    public Outcall withConfiguredTimeout(@Nonnull String configKey) {
+        Duration connectTimeout =
```

Comment:
We should probably get an extension like ext = Sirius.getSettings().getExtension("http.outcall.timeouts", configKey);
ext.get(...) 

Thisway, defaults of the default section will apply and the check below can be removed

Reply:
suggestions are now implemented